tests/heavy-tests/README.md

```diff
@@ -0,0 +1,4 @@
+# Heavy Tests Subdirectory
+
+This directory encompasses all the comprehensive tests for the project. It executes all project computations utilizing a
+range of technologies to guarantee the seamless functionality of every aspect.
```

Comment:
This directory includes all the heavy tests for the project. It executes all project testing examples using a different set of configurations to guarantee the seamless functionality of every test.